Dockerfile

```diff
@@ -1,24 +1,21 @@
-FROM registry.access.redhat.com/ubi8/python-38
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest AS base
```

Comment:
@abaiken, is it not better to force the number of the version to avoid future unexpected changes?

Reply:
It likely is better for consistency to specify the version - I pushed up a change 😁